### ISSUE TITLE
VoiceOver does not announce times correctly

### ISSUE THREAD

**Issue by Auroratide** [EmateaGithub/components#1]
VoiceOver announces `em-time` components very inconsistently. For instance, I've observed it saying "2086-05-29" as both the correct "May Twenty-Ninth, Two thousand eighty six", and the very incorrect "Two thousand eighty-six to two thousand five dash twenty nine".

* Investigate what NVDA does with this
* Either something about the component's structure needs to change, or
* ARIA needs to be used to in some sense manually tell the screen reader what to say.
